src/toil/test/provisioners/clusterScalerTest.py

```diff
@@ -170,7 +208,7 @@ def testClusterScaling(self):
         config.maxNodes = [10]
 
         # Algorithm parameters
-        config.alphaPacking = 0.0
+        config.alphaTime = 3600
```

Comment:
I kept these tests at 3600 because they were written I had 3600 as the default. They probably still pass with 1800, but I haven't tried.

(It wouldn't be a bad thing if they didn't pass with 1800. Some of these tests are testing specific behavior of the bin-packing algorithm with a fixed value of alpha. I think for those tests I made sure that any remotely reasonable value of alpha [less than a day or so] would do, but I'm not 100% sure.)